### PR TITLE
Documentation improved

### PR DESCRIPTION
**Pull request by Kaosfaktor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Simple Counter in React
 
-## Start a new Prohect in React
-You need NodeJS und NPM
+## Start a new Project in React, you need NodeJS and NPM
```
Comment (flonix8) [inline README.md:3]: This goes for all section headings: 
I would keep them shorter and put the sentences into the section...

File: README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+Node.js is a Javascript runtime environment that processes data very quickly and is easy to scale. NPM is a Package Manager which is available for public use. NPM allows us to easily use code written by others without having to write it ourselves during development.
```
Comment (flonix8) [inline README.md:4]: I would suggest less marketing but maybe some information on how to actually get `npm` installed, i.e. that you have to install `node` to get it. Mentioning that npm is a JavaScript package manager that is formally part of the `node` ecosystem should be enough.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
 
-### We are creating a new React project with NPX
+### We are creating a new React project with NPX.
+NPX stands for Node Package Execute. NPX comes with NPM, when NPM is installed above the 5.2.0 version, it gets installed automatically. NPX is an NPM package runner and its role is to execute the package from the registry without even installing that package.
 ```
 npx create-react-app simple-counter
```
Comment (flonix8) [inline README.md:9]: create-react-app is sadly deprecated. This is why we had to remove it from all our frontends. If it is okay for a training repository is another question.

File: README.md [hunk 4]
```diff
@@ -14,7 +15,7 @@ cd simple-counter
 code .
 ```
 
-### Start the application now with npm start (runs on localhost:3000), here we can view our result "on-the-fly"
+### Start the application now with npm start (runs on localhost:3000), here we can view our result "on-the-fly".
```
Comment (flonix8) [inline README.md:18]: Tell the user to point their browser at `localhost:3000`

**Review by flonix8 — CHANGES_REQUESTED** [review]

**Comment by flonix8** [comment]
The elephant in the room is probably that `create-react-app` is deprecated and should not be used anymore. I have not looked yet if there is a somewhat easy to use replacement....

File: README.md (hunk 2 above)
Comment (witchpou) [inline README.md:4]:
```suggestion
Node.js is a Javascript runtime environment that processes data very quickly and is easy to scale. NPM is a Package Manager which is available for public use (see https://www.npmjs.com/). Dependencies resolved via npm are listed in the package.json. The command `npm install` creates a folder `node-modules` in your project directory and downloads all dependencies.
```

**Review by witchpou — COMMENTED** [review]
I would suggest to make a agenda/toc and chapters like:
* creating a react app
* build and dependency management
* creating a react component
* manage form input

Add topic:
* react hooks
  ** use state
  ** update values using react hook "useEffect" (see: https://www.w3schools.com/react/react_useeffect.asp)

File: README.md [hunk 5]
```diff
@@ -38,6 +39,7 @@ export default App;
 ```
 
 ### For a simple test, we can insert the following code in line 8 and view it directly in the browser
+The 'p' element defines a paragraph of text. Paragraphs are used to structure a text.
```
Comment (witchpou) [inline README.md:42]: this is html and not relevant for react

**Pull request merged**